control-plane-operator/hostedclusterconfigoperator/controllers/resources/registry/registry.go

```diff
@@ -25,9 +25,9 @@ func ReconcileRegistryConfig(cfg *imageregistryv1.Config, platform hyperv1.Platf
 	if cfg.Spec.HTTPSecret == "" {
 		cfg.Spec.HTTPSecret = generateImageRegistrySecret()
 	}
-	if (platform == hyperv1.KubevirtPlatform || platform == hyperv1.NonePlatform) &&
-		cfg.Spec.Storage.EmptyDir == nil {
 
+	storageNotSet := cfg.Spec.Storage == imageregistryv1.ImageRegistryConfigStorage{}
+	if (platform == hyperv1.KubevirtPlatform || platform == hyperv1.NonePlatform) && storageNotSet {
```

Comment:
should "NonePlatform" be "AgentPlatform" instead?

Reply:
I'd stick to None for now unless we explicitly need to change the Agent one